src/SOLID/OpenClosedPrinciple/Error/FrontEndDeveloper.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Deg540\PHPTestingBoilerplate\SOLID\OpenClosedPrinciple\Error;
+
+class FrontEndDeveloper
+{
+    private string $name;
+    private string $program;
+
+    public function __construct(string $name, string $program)
+    {
+        $this->name = $name;
+        $this->program = $program;
+    }
+
+    public function doWork(): string
+    {
+        return $this->name . ' is developing ' . $this->program;
```

Comment:
En vez de `is developing` ya que es FrontEndDeveloper que te parece si ponemos, esta maquetando??

```suggestion
        return $this->name . ' is laying out ' . $this->program;
```